docs/guides/migration_guide_equinix_metal.md

```diff
@@ -0,0 +1,235 @@
+---
+page_title: "Migrating from the Packet provider"
```

Comment:
We have `migration_guide_packet.md` elsewhere in this PR which covers Packet -> Equinix

This guide should be "Migrating from the Equinix Metal provider", covering Metal -> Equinix.

Reply:
The changes had not been saved in my previous commit. Now the two guides are included

https://github.com/equinix/terraform-provider-equinix/pull/95/commits/4a9d30312eba66c1da9cc23b5be18749cefd7dc6